src/Obfuscatories/MissyElliottObfuscator.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace RobThree\HumanoID\Obfuscatories;
+
+class MissyElliottObfuscator implements SymmetricObfuscatorInterface
```

Comment:
Us shipping this one isn't a "must-have" for me.
However it's the best example obfuscator (we have currently) that is advanced enough to show off the feature and simple enough to grok.

That said, I do think it's a super fun one and think we should ["SEND IT"](https://www.youtube.com/watch?v=mzOUgwsQ_hM).

Reply:
I think it's really fun too. And like you said, it's good to ship a working example and I do think this is in the sweet spot of being advanced enough to be usable, but simple enough to not add a ton of complexity and extra code.